js/map_utils.js

```diff
@@ -187,7 +196,15 @@ function updateCategory(category, prevCategory) {
   updatePies(category, prevCategory);
   
   updateLegend(category);
-  
+  if(updateCategoryTimer){
+    clearTimeout(updateCategoryTimer);
+  }
+  updateCategoryTimer = setTimeout(function(){
+     gtag('event', action + ' update category', {
+  'event_category': 'figure',
+  'event_label': category + '; view=' + activeView
```

Comment:
would you want to use `prevCategory` in the event tag? Noticed you did `oldView` and `newView` in the other one.

Reply:
It seemed less valuable to me than knowing the previous view.  Although I suppose there's no reason not to include it other than my own laziness 😁